<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="registerUserButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Zarejestruj nowego użytkownika</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showPatientsButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Pokaż listę pacjentów</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerPatientButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Zarejestruj pacjenta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255); color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>Zakończ program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerVisitButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Zarejestruj wizytę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showVisitButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Pokaż listę wizyt</string>
   </property>
  </widget>
  <widget class="QLabel" name="widget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>501</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addDoctorButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Dodaj lekarza</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>registerUserButton</zorder>
  <zorder>showPatientsButton</zorder>
  <zorder>registerPatientButton</zorder>
  <zorder>endButton</zorder>
  <zorder>registerVisitButton</zorder>
  <zorder>showVisitButton</zorder>
  <zorder>addDoctorButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
